src/main/java/com/amazonaws/kinesisvideo/parser/utilities/FragmentMetadataVisitor.java

```diff
@@ -352,7 +352,7 @@ private Map<String, String> getTagNameToValueMap() {
         Map<Long, String> parentElementNumberToTagValue = tagElements.stream()
                 .filter(e -> MkvTypeInfos.TAGSTRING.equals(e.getElementMetaData().getTypeInfo()))
                 .filter(e -> MkvTypeInfos.SIMPLETAG.equals(getParentElement(e).getTypeInfo()))
-                .collect(Collectors.toMap(e -> getParentElement(e).getElementNumber(), this::getMkvElementStringVal));
+                .collect(Collectors.toMap(e -> getParentElement(e).getElementNumber(), this::getMkvElementStringVal, (a,b)->a));
```

Comment:
(a,b)->b is equivalent to [Function::Identity](https://docs.oracle.com/en%2Fjava%2Fjavase%2F11%2Fdocs%2Fapi%2F%2F/java.base/java/util/function/Function.html#identity())